tests/v2/validation/provisioning/rke2/README.md

```diff
@@ -44,7 +44,8 @@ provisioningInput is needed to the run the RKE2 tests, specifically kubernetesVe
     "nodeProviders": ["ec2"],
     "hardened": true,
     "psact": "",
-    "clusterSSHTests": ["CheckCPU"]
+    "clusterSSHTests": ["CheckCPU"],
+    "drainBeforeDelete": true
```

Comment:
The K3S README needs to be updated as well with this new field.

Reply:
Sounds good, I also added drainBeforeDelete to rke1 READMe as well.